lib/utilities/constants/app_constants.dart

```diff
@@ -44,4 +44,6 @@ Iterable<Locale> supportedLocalesList = const [
   Locale('es', 'ES'),
```

Comment:
This isn't fully your task and somehow it was missed during the previous PR's but could you please go to the 
```/lib/utilities/extensions/locale_extension.dart``` file and add these other 5 languages to the switch case block, so that it returns the appropriate language name instead of ```en_GB```.
Thanks🙏🏾